Use delete_all_pvcs method to delete PVCs

jilju opened the pull request:
Enhancement: Using delete_all_pvcs for deleting PVCs
Also , Fixes #345 
Signed-off-by: Jilju Joy <jijoy@redhat.com>

prsurve commented:
Hi @jilju as per pr #284 delete_all_pvc will be renamed to delete_pvcs(pvc_name) and need to pass pvc lists

vavuthu commented:
LGTM. Please fix the checks which was failed and paste the o/p of test case and mark it as verified.

jilju commented:
> Hi @jilju as per pr #284 delete_all_pvc will be renamed to delete_pvcs(pvc_name) and need to pass pvc lists
@prsurve Thank you for pointing this out.
I will modify the test case based on the changes in PR 284

ebenahar commented on ocs_ci/ocs/resources/pvc.py:
```diff
@@ -70,15 +70,20 @@ def resize_pvc(self, new_size, verify=False):
-        kind=constants.PVC, namespace=config.ENV_DATA['cluster_namespace']
+        kind=constants.PVC, namespace=namespace
     )
     ocp_pvc_list = get_all_pvcs()
```
need to send a namespace here?

jilju replied:
Yes, Done

jilju commented:
Output :
```
Collecting Cluster versions
=============================================================================================== test session starts ===============================================================================================
platform linux -- Python 3.7.3, pytest-5.0.0, py-1.8.0, pluggy-0.12.0
rootdir: /home/jijoy/OCS-CI-Working/ocs-ci, inifile: pytest.ini
plugins: metadata-1.8.0, html-1.21.1, reportportal-1.0.5, logger-0.5.0
collected 1 item                                                                                                                                                                                                  

tests/manage/test_pvc_concurrent_deletion_creation.py::TestMultiplePvcConcurrentDeletionCreation::test_multiple_pvc_concurrent_creation_deletion 
------------------------------------------------------------------------------------------------- live log setup --------------------------------------------------------------------------------------------------
12:44:23 - MainThread - tests.conftest - INFO - All logs located at /tmp/logs_1562310839
12:44:23 - MainThread - tests.conftest - INFO - Running OCS basic installation
12:44:23 - MainThread - ocs_ci.ocs.openshift_ops - INFO - Testing access to cluster with /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig
12:44:23 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc cluster-info
12:44:24 - MainThread - ocs_ci.ocs.openshift_ops - INFO - Access to cluster is OK!
12:44:24 - MainThread - tests.conftest - INFO - The installation is skipped because the cluster is running
12:44:24 - Dummy-1 - ocs_ci.utility.utils - INFO - Executing command: oc --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get Pod -A -o yaml
12:44:24 - Dummy-2 - ocs_ci.utility.utils - INFO - Executing command: oc --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get StorageClass -A -o yaml
12:44:24 - Dummy-3 - ocs_ci.utility.utils - INFO - Executing command: oc --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get CephFileSystem -A -o yaml
12:44:24 - Dummy-4 - ocs_ci.utility.utils - INFO - Executing command: oc --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get CephBlockPool -A -o yaml
12:44:24 - Dummy-5 - ocs_ci.utility.utils - INFO - Executing command: oc --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get PersistentVolume -A -o yaml
12:44:24 - Dummy-6 - ocs_ci.utility.utils - INFO - Executing command: oc --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get PersistentVolumeClaim -A -o yaml
12:44:24 - Dummy-7 - ocs_ci.utility.utils - INFO - Executing command: oc --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get Secret -A -o yaml
12:44:24 - Dummy-8 - ocs_ci.utility.utils - INFO - Executing command: oc --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get Namespace -A -o yaml
12:45:03 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get Pod --selector=app=rook-ceph-tools -o yaml
12:45:05 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig rsh rook-ceph-tools-76c7d559b6-ff8bw ceph auth get-key client.admin --format json-pretty
12:45:11 - MainThread - ocs_ci.ocs.resources.ocs - INFO - Adding Secret with name secret-test-0512451126
12:45:11 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig create -f /tmp/Secretg16iy89h -o yaml
12:45:13 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get Secret secret-test-0512451126 -o yaml
12:45:15 - MainThread - ocs_ci.ocs.resources.ocs - INFO - Adding CephBlockPool with name cbp-test-0512451566
12:45:15 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig create -f /tmp/CephBlockPoolhb82v983 -o yaml
12:45:17 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get CephBlockPool cbp-test-0512451566 -o yaml
12:45:19 - MainThread - tests.helpers - INFO - Verifying that block pool cbp-test-0512451566 exists
12:45:19 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get Pod --selector=app=rook-ceph-tools -o yaml
12:45:20 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig rsh rook-ceph-tools-76c7d559b6-ff8bw ceph osd lspools --format json-pretty
12:45:30 - MainThread - tests.helpers - INFO - POOLS are [{'poolnum': 1, 'poolname': 'rbd'}, {'poolnum': 2, 'poolname': 'ocsci-cephfs-metadata'}, {'poolnum': 3, 'poolname': 'ocsci-cephfs-data0'}, {'poolnum': 106, 'poolname': 'cbp-test-0512451566'}]
12:45:30 - MainThread - ocs_ci.ocs.resources.ocs - INFO - Adding StorageClass with name storageclass-test-0512453036
12:45:30 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig create -f /tmp/StorageClasswru0tq3d -o yaml
12:45:32 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get StorageClass storageclass-test-0512453036 -o yaml
12:45:33 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc new-project namespace-test-0512453392
12:45:37 - MainThread - ocs_ci.ocs.resources.pvc - INFO - Creating Persistent Volume Claim test-pvc1
12:45:37 - MainThread - ocs_ci.ocs.resources.ocs - INFO - Adding PersistentVolumeClaim with name test-pvc1
12:45:37 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0512453392 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig create -f /tmp/PersistentVolumeClaimlnyd504c -o yaml
12:45:39 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0512453392 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get PersistentVolumeClaim test-pvc1 -o yaml
12:45:40 - MainThread - ocs_ci.ocs.resources.pvc - INFO - Created Persistent Volume Claim test-pvc1
12:45:40 - MainThread - ocs_ci.ocs.resources.pvc - INFO - Creating Persistent Volume Claim test-pvc2
12:45:40 - MainThread - ocs_ci.ocs.resources.ocs - INFO - Adding PersistentVolumeClaim with name test-pvc2
12:45:40 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0512453392 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig create -f /tmp/PersistentVolumeClaim1nri300r -o yaml
12:45:42 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0512453392 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get PersistentVolumeClaim test-pvc2 -o yaml
12:45:44 - MainThread - ocs_ci.ocs.resources.pvc - INFO - Created Persistent Volume Claim test-pvc2
12:45:44 - MainThread - ocs_ci.ocs.resources.pvc - INFO - Creating Persistent Volume Claim test-pvc3
12:45:44 - MainThread - ocs_ci.ocs.resources.ocs - INFO - Adding PersistentVolumeClaim with name test-pvc3
12:45:44 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0512453392 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig create -f /tmp/PersistentVolumeClaimtv27py5p -o yaml
12:45:46 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0512453392 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get PersistentVolumeClaim test-pvc3 -o yaml
12:45:48 - MainThread - ocs_ci.ocs.resources.pvc - INFO - Created Persistent Volume Claim test-pvc3
12:45:48 - MainThread - ocs_ci.ocs.resources.pvc - INFO - Creating Persistent Volume Claim test-pvc4
12:45:48 - MainThread - ocs_ci.ocs.resources.ocs - INFO - Adding PersistentVolumeClaim with name test-pvc4
12:45:48 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0512453392 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig create -f /tmp/PersistentVolumeClaimunzmrgju -o yaml
12:45:50 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0512453392 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get PersistentVolumeClaim test-pvc4 -o yaml
12:45:51 - MainThread - ocs_ci.ocs.resources.pvc - INFO - Created Persistent Volume Claim test-pvc4
12:45:51 - MainThread - ocs_ci.ocs.resources.pvc - INFO - Creating Persistent Volume Claim test-pvc5
12:45:51 - MainThread - ocs_ci.ocs.resources.ocs - INFO - Adding PersistentVolumeClaim with name test-pvc5
12:45:51 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0512453392 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig create -f /tmp/PersistentVolumeClaim_q1xxg6w -o yaml
12:45:53 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0512453392 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get PersistentVolumeClaim test-pvc5 -o yaml
12:45:55 - MainThread - ocs_ci.ocs.resources.pvc - INFO - Created Persistent Volume Claim test-pvc5
12:45:55 - MainThread - ocs_ci.ocs.resources.pvc - INFO - Creating Persistent Volume Claim test-pvc6
12:45:55 - MainThread - ocs_ci.ocs.resources.ocs - INFO - Adding PersistentVolumeClaim with name test-pvc6
12:45:55 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0512453392 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig create -f /tmp/PersistentVolumeClaimyr050zwb -o yaml
12:45:57 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0512453392 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get PersistentVolumeClaim test-pvc6 -o yaml
12:45:59 - MainThread - ocs_ci.ocs.resources.pvc - INFO - Created Persistent Volume Claim test-pvc6
12:45:59 - MainThread - ocs_ci.ocs.resources.pvc - INFO - Creating Persistent Volume Claim test-pvc7
12:45:59 - MainThread - ocs_ci.ocs.resources.ocs - INFO - Adding PersistentVolumeClaim with name test-pvc7
12:45:59 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0512453392 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig create -f /tmp/PersistentVolumeClaimllx8pe09 -o yaml
12:46:00 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0512453392 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get PersistentVolumeClaim test-pvc7 -o yaml
12:46:02 - MainThread - ocs_ci.ocs.resources.pvc - INFO - Created Persistent Volume Claim test-pvc7
12:46:02 - MainThread - ocs_ci.ocs.resources.pvc - INFO - Creating Persistent Volume Claim test-pvc8
12:46:02 - MainThread - ocs_ci.ocs.resources.ocs - INFO - Adding PersistentVolumeClaim with name test-pvc8
12:46:02 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0512453392 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig create -f /tmp/PersistentVolumeClaimema_w3ye -o yaml
12:46:04 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0512453392 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get PersistentVolumeClaim test-pvc8 -o yaml
12:46:06 - MainThread - ocs_ci.ocs.resources.pvc - INFO - Created Persistent Volume Claim test-pvc8
12:46:06 - MainThread - ocs_ci.ocs.resources.pvc - INFO - Creating Persistent Volume Claim test-pvc9
12:46:06 - MainThread - ocs_ci.ocs.resources.ocs - INFO - Adding PersistentVolumeClaim with name test-pvc9
12:46:06 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0512453392 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig create -f /tmp/PersistentVolumeClaim120w4757 -o yaml
12:46:08 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0512453392 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get PersistentVolumeClaim test-pvc9 -o yaml
12:46:10 - MainThread - ocs_ci.ocs.resources.pvc - INFO - Created Persistent Volume Claim test-pvc9
12:46:10 - MainThread - ocs_ci.ocs.resources.pvc - INFO - Creating Persistent Volume Claim test-pvc10
12:46:10 - MainThread - ocs_ci.ocs.resources.ocs - INFO - Adding PersistentVolumeClaim with name test-pvc10
12:46:10 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0512453392 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig create -f /tmp/PersistentVolumeClaimr5q94stn -o yaml
12:46:12 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0512453392 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get PersistentVolumeClaim test-pvc10 -o yaml
12:46:13 - MainThread - ocs_ci.ocs.resources.pvc - INFO - Created Persistent Volume Claim test-pvc10
12:46:13 - MainThread - tests.manage.test_pvc_concurrent_deletion_creation - INFO - Created initial 10 PVCs
12:46:13 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0512453392 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get PersistentVolumeClaim test-pvc1 -o yaml
12:46:15 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0512453392 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get PersistentVolumeClaim test-pvc2 -o yaml
12:46:17 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0512453392 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get PersistentVolumeClaim test-pvc3 -o yaml
12:46:18 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0512453392 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get PersistentVolumeClaim test-pvc4 -o yaml
12:46:20 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0512453392 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get PersistentVolumeClaim test-pvc5 -o yaml
12:46:27 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0512453392 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get PersistentVolumeClaim test-pvc6 -o yaml
12:46:29 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0512453392 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get PersistentVolumeClaim test-pvc7 -o yaml
12:46:31 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0512453392 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get PersistentVolumeClaim test-pvc8 -o yaml
12:46:32 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0512453392 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get PersistentVolumeClaim test-pvc9 -o yaml
12:46:34 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0512453392 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get PersistentVolumeClaim test-pvc10 -o yaml
12:46:35 - MainThread - tests.manage.test_pvc_concurrent_deletion_creation - INFO - Initial 10 PVCs are in Bound state
-------------------------------------------------------------------------------------------------- live log call --------------------------------------------------------------------------------------------------
12:46:35 - MainThread - ocs_ci.utility.utils - INFO - Executing command: for i in `seq 1 10`;do oc delete pvc test-pvc$i -n namespace-test-0512453392;done
12:46:35 - MainThread - ocs_ci.ocs.resources.pvc - INFO - Creating Persistent Volume Claim test-pvc-re1
12:46:35 - MainThread - ocs_ci.ocs.resources.ocs - INFO - Adding PersistentVolumeClaim with name test-pvc-re1
12:46:35 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0512453392 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig create -f /tmp/PersistentVolumeClaimc1ydv0wu -o yaml
12:46:37 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0512453392 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get PersistentVolumeClaim test-pvc-re1 -o yaml
12:46:38 - MainThread - ocs_ci.ocs.resources.pvc - INFO - Created Persistent Volume Claim test-pvc-re1
12:46:38 - MainThread - ocs_ci.ocs.resources.pvc - INFO - Creating Persistent Volume Claim test-pvc-re2
12:46:38 - MainThread - ocs_ci.ocs.resources.ocs - INFO - Adding PersistentVolumeClaim with name test-pvc-re2
12:46:38 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0512453392 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig create -f /tmp/PersistentVolumeClaimvsf7hvpl -o yaml
12:46:40 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0512453392 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get PersistentVolumeClaim test-pvc-re2 -o yaml
12:46:42 - MainThread - ocs_ci.ocs.resources.pvc - INFO - Created Persistent Volume Claim test-pvc-re2
12:46:42 - MainThread - ocs_ci.ocs.resources.pvc - INFO - Creating Persistent Volume Claim test-pvc-re3
12:46:42 - MainThread - ocs_ci.ocs.resources.ocs - INFO - Adding PersistentVolumeClaim with name test-pvc-re3
12:46:42 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0512453392 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig create -f /tmp/PersistentVolumeClaimr2ovmh4k -o yaml
12:46:44 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0512453392 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get PersistentVolumeClaim test-pvc-re3 -o yaml
12:46:45 - MainThread - ocs_ci.ocs.resources.pvc - INFO - Created Persistent Volume Claim test-pvc-re3
12:46:45 - MainThread - ocs_ci.ocs.resources.pvc - INFO - Creating Persistent Volume Claim test-pvc-re4
12:46:45 - MainThread - ocs_ci.ocs.resources.ocs - INFO - Adding PersistentVolumeClaim with name test-pvc-re4
12:46:45 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0512453392 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig create -f /tmp/PersistentVolumeClaimtak0vm61 -o yaml
12:46:47 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0512453392 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get PersistentVolumeClaim test-pvc-re4 -o yaml
12:46:48 - MainThread - ocs_ci.ocs.resources.pvc - INFO - Created Persistent Volume Claim test-pvc-re4
12:46:48 - MainThread - ocs_ci.ocs.resources.pvc - INFO - Creating Persistent Volume Claim test-pvc-re5
12:46:48 - MainThread - ocs_ci.ocs.resources.ocs - INFO - Adding PersistentVolumeClaim with name test-pvc-re5
12:46:48 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0512453392 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig create -f /tmp/PersistentVolumeClaimiv_cx0z0 -o yaml
12:46:50 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0512453392 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get PersistentVolumeClaim test-pvc-re5 -o yaml
12:46:51 - MainThread - ocs_ci.ocs.resources.pvc - INFO - Created Persistent Volume Claim test-pvc-re5
12:46:51 - MainThread - ocs_ci.ocs.resources.pvc - INFO - Creating Persistent Volume Claim test-pvc-re6
12:46:51 - MainThread - ocs_ci.ocs.resources.ocs - INFO - Adding PersistentVolumeClaim with name test-pvc-re6
12:46:51 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0512453392 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig create -f /tmp/PersistentVolumeClaimy_s55d_1 -o yaml
12:46:53 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0512453392 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get PersistentVolumeClaim test-pvc-re6 -o yaml
12:46:55 - MainThread - ocs_ci.ocs.resources.pvc - INFO - Created Persistent Volume Claim test-pvc-re6
12:46:55 - MainThread - ocs_ci.ocs.resources.pvc - INFO - Creating Persistent Volume Claim test-pvc-re7
12:46:55 - MainThread - ocs_ci.ocs.resources.ocs - INFO - Adding PersistentVolumeClaim with name test-pvc-re7
12:46:55 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0512453392 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig create -f /tmp/PersistentVolumeClaim1ptkvc4n -o yaml
12:46:57 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0512453392 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get PersistentVolumeClaim test-pvc-re7 -o yaml
12:46:58 - MainThread - ocs_ci.ocs.resources.pvc - INFO - Created Persistent Volume Claim test-pvc-re7
12:46:58 - MainThread - ocs_ci.ocs.resources.pvc - INFO - Creating Persistent Volume Claim test-pvc-re8
12:46:58 - MainThread - ocs_ci.ocs.resources.ocs - INFO - Adding PersistentVolumeClaim with name test-pvc-re8
12:46:58 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0512453392 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig create -f /tmp/PersistentVolumeClaimsq6tgnr2 -o yaml
12:47:00 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0512453392 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get PersistentVolumeClaim test-pvc-re8 -o yaml
12:47:01 - MainThread - ocs_ci.ocs.resources.pvc - INFO - Created Persistent Volume Claim test-pvc-re8
12:47:01 - MainThread - ocs_ci.ocs.resources.pvc - INFO - Creating Persistent Volume Claim test-pvc-re9
12:47:01 - MainThread - ocs_ci.ocs.resources.ocs - INFO - Adding PersistentVolumeClaim with name test-pvc-re9
12:47:01 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0512453392 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig create -f /tmp/PersistentVolumeClaim1y7fwrsl -o yaml
12:47:03 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0512453392 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get PersistentVolumeClaim test-pvc-re9 -o yaml
12:47:04 - MainThread - ocs_ci.ocs.resources.pvc - INFO - Created Persistent Volume Claim test-pvc-re9
12:47:04 - MainThread - ocs_ci.ocs.resources.pvc - INFO - Creating Persistent Volume Claim test-pvc-re10
12:47:04 - MainThread - ocs_ci.ocs.resources.ocs - INFO - Adding PersistentVolumeClaim with name test-pvc-re10
12:47:04 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0512453392 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig create -f /tmp/PersistentVolumeClaiml2dt7nk0 -o yaml
12:47:06 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0512453392 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get PersistentVolumeClaim test-pvc-re10 -o yaml
12:47:09 - MainThread - ocs_ci.ocs.resources.pvc - INFO - Created Persistent Volume Claim test-pvc-re10
12:47:09 - MainThread - tests.manage.test_pvc_concurrent_deletion_creation - INFO - Created 10 new PVCs.
12:47:09 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0512453392 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get PersistentVolumeClaim test-pvc-re1 -o yaml
12:47:11 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0512453392 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get PersistentVolumeClaim test-pvc-re2 -o yaml
12:47:13 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0512453392 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get PersistentVolumeClaim test-pvc-re3 -o yaml
12:47:14 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0512453392 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get PersistentVolumeClaim test-pvc-re4 -o yaml
12:47:16 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0512453392 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get PersistentVolumeClaim test-pvc-re5 -o yaml
12:47:18 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0512453392 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get PersistentVolumeClaim test-pvc-re6 -o yaml
12:47:19 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0512453392 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get PersistentVolumeClaim test-pvc-re7 -o yaml
12:47:21 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0512453392 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get PersistentVolumeClaim test-pvc-re8 -o yaml
12:47:22 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0512453392 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get PersistentVolumeClaim test-pvc-re9 -o yaml
12:47:24 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0512453392 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get PersistentVolumeClaim test-pvc-re10 -o yaml
12:47:25 - MainThread - tests.manage.test_pvc_concurrent_deletion_creation - INFO - Verified: Newly created PVCs are in Bound state.
12:47:25 - MainThread - tests.manage.test_pvc_concurrent_deletion_creation - INFO - Return values of command: for i in `seq 1 10`;do oc delete pvc test-pvc$i -n namespace-test-0512453392;done.
                                                                                    retcode:0
                                                                                    stdout:persistentvolumeclaim "test-pvc1" deleted
                                                                                    persistentvolumeclaim "test-pvc2" deleted
                                                                                    persistentvolumeclaim "test-pvc3" deleted
                                                                                    persistentvolumeclaim "test-pvc4" deleted
                                                                                    persistentvolumeclaim "test-pvc5" deleted
                                                                                    persistentvolumeclaim "test-pvc6" deleted
                                                                                    persistentvolumeclaim "test-pvc7" deleted
                                                                                    persistentvolumeclaim "test-pvc8" deleted
                                                                                    persistentvolumeclaim "test-pvc9" deleted
                                                                                    persistentvolumeclaim "test-pvc10" deleted
                                                                                    
                                                                                    stderr:None
12:47:25 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0512453392 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get PersistentVolumeClaim test-pvc1 -o yaml
12:47:28 - MainThread - tests.manage.test_pvc_concurrent_deletion_creation - INFO - Expected: PVC test-pvc1 does not exists 
12:47:28 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0512453392 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get PersistentVolumeClaim test-pvc2 -o yaml
12:47:30 - MainThread - tests.manage.test_pvc_concurrent_deletion_creation - INFO - Expected: PVC test-pvc2 does not exists 
12:47:30 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0512453392 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get PersistentVolumeClaim test-pvc3 -o yaml
12:47:33 - MainThread - tests.manage.test_pvc_concurrent_deletion_creation - INFO - Expected: PVC test-pvc3 does not exists 
12:47:33 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0512453392 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get PersistentVolumeClaim test-pvc4 -o yaml
12:47:34 - MainThread - tests.manage.test_pvc_concurrent_deletion_creation - INFO - Expected: PVC test-pvc4 does not exists 
12:47:34 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0512453392 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get PersistentVolumeClaim test-pvc5 -o yaml
12:47:36 - MainThread - tests.manage.test_pvc_concurrent_deletion_creation - INFO - Expected: PVC test-pvc5 does not exists 
12:47:36 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0512453392 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get PersistentVolumeClaim test-pvc6 -o yaml
12:47:38 - MainThread - tests.manage.test_pvc_concurrent_deletion_creation - INFO - Expected: PVC test-pvc6 does not exists 
12:47:38 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0512453392 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get PersistentVolumeClaim test-pvc7 -o yaml
12:47:41 - MainThread - tests.manage.test_pvc_concurrent_deletion_creation - INFO - Expected: PVC test-pvc7 does not exists 
12:47:41 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0512453392 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get PersistentVolumeClaim test-pvc8 -o yaml
12:47:42 - MainThread - tests.manage.test_pvc_concurrent_deletion_creation - INFO - Expected: PVC test-pvc8 does not exists 
12:47:42 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0512453392 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get PersistentVolumeClaim test-pvc9 -o yaml
12:47:45 - MainThread - tests.manage.test_pvc_concurrent_deletion_creation - INFO - Expected: PVC test-pvc9 does not exists 
12:47:45 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0512453392 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get PersistentVolumeClaim test-pvc10 -o yaml
12:47:47 - MainThread - tests.manage.test_pvc_concurrent_deletion_creation - INFO - Expected: PVC test-pvc10 does not exists 
12:47:47 - MainThread - tests.manage.test_pvc_concurrent_deletion_creation - INFO - Successfully deleted initial 10 PVCs
12:47:47 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get Pod --selector=app=rook-ceph-tools -o yaml
12:47:49 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig rsh rook-ceph-tools-76c7d559b6-ff8bw rbd ls -p cbp-test-0512451566 --format json
12:47:53 - MainThread - tests.manage.test_pvc_concurrent_deletion_creation - INFO - Verified: PVs associated with deleted PVCs are also deleted
PASSED                                                                                                                                                                                                      [100%]
------------------------------------------------------------------------------------------------ live log teardown ------------------------------------------------------------------------------------------------
12:47:53 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0512453392 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get PersistentVolumeClaim  -o yaml
12:47:55 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0512453392 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig delete PersistentVolumeClaim test-pvc-re1
12:48:02 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0512453392 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig delete PersistentVolumeClaim test-pvc-re10
12:48:04 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0512453392 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig delete PersistentVolumeClaim test-pvc-re2
12:48:06 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0512453392 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig delete PersistentVolumeClaim test-pvc-re3
12:48:08 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0512453392 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig delete PersistentVolumeClaim test-pvc-re4
12:48:10 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0512453392 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig delete PersistentVolumeClaim test-pvc-re5
12:48:12 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0512453392 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig delete PersistentVolumeClaim test-pvc-re6
12:48:14 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0512453392 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig delete PersistentVolumeClaim test-pvc-re7
12:48:16 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0512453392 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig delete PersistentVolumeClaim test-pvc-re8
12:48:18 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0512453392 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig delete PersistentVolumeClaim test-pvc-re9
12:48:20 - MainThread - tests.manage.test_pvc_concurrent_deletion_creation - INFO - Newly created 10 PVCs are now deleted.
12:48:20 - MainThread - ocs_ci.ocs.ocp - INFO - Switching to project openshift-storage
12:48:20 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc project openshift-storage
12:48:22 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0512453392 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig delete Project namespace-test-0512453392
12:48:25 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get StorageClass storageclass-test-0512453036 -o yaml
12:48:27 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig delete StorageClass storageclass-test-0512453036
12:48:30 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get CephBlockPool cbp-test-0512451566 -o yaml
12:48:32 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig delete CephBlockPool cbp-test-0512451566
12:48:34 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get Secret secret-test-0512451126 -o yaml
12:48:35 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig delete Secret secret-test-0512451126
12:48:37 - Dummy-7 - ocs_ci.utility.utils - INFO - Executing command: oc --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get Pod -A -o yaml
12:48:37 - Dummy-9 - ocs_ci.utility.utils - INFO - Executing command: oc --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get StorageClass -A -o yaml
12:48:37 - Dummy-2 - ocs_ci.utility.utils - INFO - Executing command: oc --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get CephFileSystem -A -o yaml
12:48:37 - Dummy-10 - ocs_ci.utility.utils - INFO - Executing command: oc --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get CephBlockPool -A -o yaml
12:48:37 - Dummy-1 - ocs_ci.utility.utils - INFO - Executing command: oc --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get PersistentVolume -A -o yaml
12:48:37 - Dummy-5 - ocs_ci.utility.utils - INFO - Executing command: oc --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get PersistentVolumeClaim -A -o yaml
12:48:37 - Dummy-8 - ocs_ci.utility.utils - INFO - Executing command: oc --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get Secret -A -o yaml
12:48:37 - Dummy-11 - ocs_ci.utility.utils - INFO - Executing command: oc --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get Namespace -A -o yaml


=========================================================================================== 1 passed in 297.41 seconds ============================================================================================
```

jilju commented:
> > Hi @jilju as per pr #284 delete_all_pvc will be renamed to delete_pvcs(pvc_name) and need to pass pvc lists
> > @prsurve Thank you for pointing this out.
> > I will modify the test case based on the changes in PR 284
This PR also have the fix for issue #345 (high priority). The changes in the PR 284 will be incorporated later in this test case once PR 284 is merged.
